### ISSUE TITLE
Unexpected input(s) 'changesNotSentForReview', valid inputs are ['serviceAccountJson', 'serviceAccountJsonPlainText', 'packageName', 'releaseFile', 'releaseFiles', 'releaseName', 'track', 'inAppUpdatePriority', 'userFraction', 'status', 'whatsNewDirectory', 'mappingFile']

### ISSUE THREAD

**Issue by youknow-yoonho** [r0adkll/upload-google-play#123]
**Describe the bug**

`changesNotSentForReview` is not working...

**Workflow Step Configuration**

```yml
- name: Deploy Production
  uses: r0adkll/upload-google-play@v1
  with:
    serviceAccountJsonPlainText: ***
    packageName: my.app
    releaseFiles: app/build/outputs/bundle/release/app-release.aab
    track: internal
    status: completed
    whatsNewDirectory: distribution/whatsnew
    changesNotSentForReview: true
```

**Step Debugging**

- [ ] I have enabled [Step Debug Logging](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/enabling-debug-logging#enabling-step-debug-logging)

```
Creating a new Edit for this release
Validating track 'internal'
Uploading app/build/outputs/bundle/release/app-release.aab
Successfully uploaded 1 artifacts
Adding 1 artifacts to release on 'internal' track
Committing the Edit
Error: Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.
```

**Comment by ThomasAunvik** [comment]
'changesNotSentForReview' is only documented on current master branch, and isn't in the latest v1 release. use @master. Or wait till next release?

**Comment by boswelja** [comment]
Unfortunately that is the correct answer. I was planning on doing a release a while ago but got sidetracked 😅

**Comment by boswelja** [comment]
This should now be available, as of v1.0.18
